less/cards.less

```diff
@@ -16,6 +23,18 @@
   margin-bottom: 15px;
   outline: none;
   cursor:pointer;
+  
+  &:active {
```

Comment:
What is this for? I see no difference when clicking the item, because there is already no outline for `.post` on line 17/24

Reply:
Copy past methology ... I have removed this block